dependencies.properties

```diff
@@ -288,34 +288,33 @@ org.slf4j\:slf4j-api=1.7.11
 org.slf4j\:slf4j-jdk14=1.7.11
 org.slf4j\:slf4j-log4j12=1.7.11
 org.spockframework\:spock-core=0.7-groovy-2.0
-org.springframework\:spring-aop=4.1.6.RELEASE
-org.springframework\:spring-aspects=4.1.6.RELEASE
-org.springframework\:spring-beans=4.1.6.RELEASE
-org.springframework\:spring-context=4.1.6.RELEASE
-org.springframework\:spring-context-support=4.1.6.RELEASE
-org.springframework\:spring-core=4.1.6.RELEASE
-org.springframework\:spring-expression=4.1.6.RELEASE
-org.springframework\:spring-instrument=4.1.6.RELEASE
-org.springframework\:spring-instrument-tomcat=4.1.6.RELEASE
-org.springframework\:spring-jdbc=4.1.6.RELEASE
-org.springframework\:spring-jms=4.1.6.RELEASE
+org.springframework\:spring-aop=4.2.0.RC2
+org.springframework\:spring-aspects=4.2.0.RC2
+org.springframework\:spring-beans=4.2.0.RC2
+org.springframework\:spring-context=4.2.0.RC2
+org.springframework\:spring-context-support=4.2.0.RC2
+org.springframework\:spring-core=4.2.0.RC2
+org.springframework\:spring-expression=4.2.0.RC2
+org.springframework\:spring-instrument=4.2.0.RC2
+org.springframework\:spring-instrument-tomcat=4.2.0.RC2
+org.springframework\:spring-jdbc=4.2.0.RC2
+org.springframework\:spring-jms=4.2.0.RC2
 org.springframework\:springloaded=1.2.0.RELEASE
-org.springframework\:spring-messaging=4.1.6.RELEASE
-org.springframework\:spring-orm=4.1.6.RELEASE
-org.springframework\:spring-oxm=4.1.6.RELEASE
-org.springframework\:spring-test=4.1.6.RELEASE
-org.springframework\:spring-tx=4.1.6.RELEASE
-org.springframework\:spring-web=4.1.6.RELEASE
-org.springframework\:spring-webmvc=4.1.6.RELEASE
-org.springframework\:spring-webmvc-portlet=4.1.6.RELEASE
-org.springframework\:spring-websocket=4.1.6.RELEASE
-org.springframework.amqp\:spring-amqp=1.4.5.RELEASE
-org.springframework.amqp\:spring-erlang=1.4.5.RELEASE
```

Comment:
Excellent, was just checking, as this jumped at me while merging.

On Mon, Jul 13, 2015 at 6:53 PM, Gary Russell <notifications@github.com>
wrote:

> In dependencies.properties
> <https://github.com/spring-projects/spring-xd/pull/1736#discussion_r34484252>
> :
>
> >  org.springframework\:springloaded=1.2.0.RELEASE
> > -org.springframework\:spring-messaging=4.1.6.RELEASE
> > -org.springframework\:spring-orm=4.1.6.RELEASE
> > -org.springframework\:spring-oxm=4.1.6.RELEASE
> > -org.springframework\:spring-test=4.1.6.RELEASE
> > -org.springframework\:spring-tx=4.1.6.RELEASE
> > -org.springframework\:spring-web=4.1.6.RELEASE
> > -org.springframework\:spring-webmvc=4.1.6.RELEASE
> > -org.springframework\:spring-webmvc-portlet=4.1.6.RELEASE
> > -org.springframework\:spring-websocket=4.1.6.RELEASE
> > -org.springframework.amqp\:spring-amqp=1.4.5.RELEASE
> > -org.springframework.amqp\:spring-erlang=1.4.5.RELEASE
>
> Yes, spring-erlang is no more (as of 1.5.0). It is replaced by the use of
> the rabbit REST API via the RabbitManagementTemplate.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/spring-projects/spring-xd/pull/1736/files#r34484252>.
>